edge-apps/weather/index.html

```diff
@@ -48,24 +48,40 @@
                 </div>
             </section>
         </div>
-        <!--removeIf(sentry)-->
-        <script src="https://js.sentry-cdn.com/SENTRY_ID.min.js" crossorigin="anonymous"></script>
-        <!--endRemoveIf(sentry)-->
 
-        <!--removeIf(googleAnalytics)-->
-        <script async src="https://www.googletagmanager.com/gtag/js?id=GA_API_KEY"></script>
+        <script src="screenly.js?version=1"></script>
+
         <script>
+            const sentryId = screenly.settings.sentry_id;
+            const gaApiKey = screenly.settings.ga_api_key;
+
+            const sentryScript = document.createElement('script');
+            sentryScript.src = `https://js.sentry-cdn.com/${sentryId}.min.js`;
+            sentryScript.crossOrigin = 'anonymous';
+            document.head.appendChild(sentryScript);
+
+            const gtagScript = document.createElement('script');
+            gtagScript.src = `https://www.googletagmanager.com/gtag/js?id=${gaApiKey}`;
+            gtagScript.async = true;
+            document.head.appendChild(gtagScript);
+
             window.dataLayer = window.dataLayer || [];
             function gtag(){dataLayer.push(arguments);}
-            gtag('js', new Date());
 
-            gtag('config', 'GA_API_KEY');
+            gtag('js', new Date());
+            gtag('config', gaApiKey);
         </script>
-        <!--endRemoveIf(googleAnalytics)-->
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.4/moment-with-locales.min.js" integrity="sha512-42PE0rd+wZ2hNXftlM78BSehIGzezNeQuzihiBCvUEB3CVxHvsShF86wBWwQORNxNINlBPuq7rG4WWhNiTVHFg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+        <script
+            src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.4/moment-with-locales.min.js"
+            integrity="sha512-42PE0rd+wZ2hNXftlM78BSehIGzezNeQuzihiBCvUEB3CVxHvsShF86wBWwQORNxNINlBPuq7rG4WWhNiTVHFg=="
+            crossorigin="anonymous"
+            referrerpolicy="no-referrer"
+        ></script>
         <script src="/static/js/icons.js"></script>
-        <script src="/static/js/main.js"></script>
-        <script src="/static/js/load-metadata.js"></script>
-        <script src="/static/js/load-screenly.js"></script>
+        <script src="/static/js/main.js">
+            console.log('Screenly OSE Weather App')
```

Comment:
Does this work? There's an src specified along with inline script within the script tags.

Reply:
Thanks for pointing this one out. I tried it again, and yes, it doesn't work. I'll go remove it in the succeeding commits.

Reply:
Just got back here after addressing code review comments for the Clock migration. My bad, that's just a temporary change.